tinygrad/tensor.py

```diff
@@ -579,7 +580,7 @@ def __matmul__(self, x) -> Tensor: return self.matmul(x)
   def __radd__(self, x) -> Tensor: return self.add(x, True)
   def __rsub__(self, x) -> Tensor: return self.sub(x, True)
   def __rmul__(self, x) -> Tensor: return self.mul(x, True)
-  def __rpow__(self, x) -> Tensor: return self.pow(x, True)
+  def __rpow__(self, x) -> Tensor: return Tensor(x, requires_grad=False).pow(self)
```

Comment:
This shouldn't be needed, why doesn't pow match?

Reply:
This only gets called when you do float**Tensor. Instead of having a "reverse" argument, I just convert the float to a Tensor and then run pow. I don't know if there is a way to get the log of a float without converting it to a Tensor.

Reply:
umm, `math.log`?

Reply:
I wanted to do it on the GPU but I guess it doesn't matter